src/pages/resultsView/download/DownloadUtils.ts

```diff
@@ -458,4 +476,25 @@ export function hasValidMutationData(sampleAlterationDataByGene: {[key: string]:
 function extractMutationValue(alteration: ExtendedAlteration)
 {
     return alteration.proteinChange;
+}
+
+export function decideMolecularProfileSortingOrder(profileType: MolecularProfile['molecularAlterationType']) {
+    switch (profileType) {
+        case "MUTATION_EXTENDED":
+            return 1;
+        case "COPY_NUMBER_ALTERATION":
+            return 2;
+        case "GENESET_SCORE":
+            return 3;
+        case "MRNA_EXPRESSION":
+            return 4;
+        case "METHYLATION":
+            return 5;
+        case "METHYLATION_BINARY":
+            return 6;
+        case "PROTEIN_LEVEL":
+            return 7;
+        default:
+            return Number.MAX_VALUE;
+    }
```

Comment:
sort the molecular profiles as we did in the [MolecularProfileSelector](https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/components/query/MolecularProfileSelector.tsx#L40-L46)